sky/utils/common_utils.py

```diff
@@ -228,6 +232,13 @@ def get_global_job_id(job_timestamp: str,
     return global_job_id
 
 
+def get_unique_task_name(_: 'task_lib.Task') -> str:
```

Comment:
```suggestion
def get_unique_task_name() -> str:
```
if we are not using this argument, why are we still keeping it?